003-loginwithpom/src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,68 @@
+package tests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Properties;
+
+public class LoginTest {
+    private WebDriver driver;
+    private LoginPage loginPage;
+    private Properties properties;
+    private WebDriverWait wait;
+
+    @BeforeClass
+    public void setup() throws IOException {
+        // Load config.properties
+        properties = new Properties();
+        FileInputStream configStream = new FileInputStream("src\\test\\java\\tests\\config.properties");
+        properties.load(configStream);
+
+        // Initialize WebDriver
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+
+        // Implicit Wait
+        int implicitWait = Integer.parseInt(properties.getProperty("implicit.wait.duration"));
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(implicitWait));
+
+        // Initialize Page Object
+        loginPage = new LoginPage(driver);
+
+        // WebDriverWait
+        int assertionWait = Integer.parseInt(properties.getProperty("assertion.wait.duration"));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(assertionWait));
+    }
+
+    @Test
+    public void testValidLogin() {
```

Comment:
create a method for it,    may be link   loginOrangeHRM(username,password)

Reply:
once you write method... call that method here !

Reply:
done changes in LoginPage class added login method and also update LoginTest to use login method